person.rb

```diff
@@ -24,5 +25,11 @@ def can_use_services?
   def correct_name
     @name
   end
+
+  def add_rental(rental)
+    @rentals.push(rental)
+    rental.person = self
+  end
```

Comment:
- [ ] the add_rental method should take two parameters(as it is in your Rental class), date and book so as to create a new Rental using self as the third parameter(person), kindly fix this
```ruby
 def add_rental(date, book)
    Rental.new(date, book, self)
  end
```

Reply:
Hello @VICTOR-CHUKWU. I have a doubt. The rental that I will receive as an argument when I can the *add_rental* method won't be an instantiated object of class Rental?
Like this:
```ruby
rental = Rental.new(date, book, person)

add_rental(rental)

```

If so, I would not have to have *date* and *book* as arguments in the add_rental method.
Please let me know if my approach is the right one or no.

Reply:
yes you are creating a new instance of Rental, hope the zoom call helped :+1:

Reply:
Thank you for clarifying @VICTOR-CHUKWU.